prototype_source/prototype_index.rst

```diff
@@ -207,7 +207,9 @@ Prototype features are not available as part of binary distributions like PyPI o
    prototype/fx_graph_mode_ptq_dynamic.html
    prototype/fx_graph_mode_ptq_static.html
    prototype/graph_mode_dynamic_bert_tutorial.html
-   prototype/quantization_in_pytorch_2_0_export_tutorial.html
+   prototype/quantization_in_pytorch_2_export_tutorial.html
```

Comment:
this needs to be updated as well, to pt2e_quantizer.html

Reply:
oh I see. I misunderstood your previous comment